src/net/slashie/expedition/ui/oryx/ExpeditionContinuousGFXUISelector.java

```diff
@@ -203,34 +234,40 @@ public void activate() {
 		chopButton.setEnabled(true);
 		anchorButton.setEnabled(true);
 		searchAroundButton.setEnabled(true);
-		musicButton.setEnabled(true);
-		sfxButton.setEnabled(true);
-		saveButton.setEnabled(true);
-		quitButton.setEnabled(true);
-		logButton.setEnabled(true);
-		
+		/*
+		 * musicButton.setEnabled(true); sfxButton.setEnabled(true);
+		 * saveButton.setEnabled(true); quitButton.setEnabled(true);
+		 * logButton.setEnabled(true);
+		 */
+		commandPanel.setEnabled(true);
 		// Make buttons visible
+		/* sfxButton.setVisible(true); */
 		buttonsPanel.setVisible(true);
-		musicButton.setVisible(true);
-		sfxButton.setVisible(true);
-		saveButton.setVisible(true);
-		quitButton.setVisible(true);
-		logButton.setVisible(true);
+		/*
+		 * musicButton.setVisible(true); sfxButton.setVisible(true);
+		 * saveButton.setVisible(true); quitButton.setVisible(true);
+		 * logButton.setVisible(true);
+		 */
+		commandPanel.setVisible(true);
 		updateButtonStatus();
 	}
-	
-	public void shutdown() {
+
+	public void shutdown()
+	{
 		super.shutdown();
 		buttonsPanel.setVisible(false);
-		musicButton.setVisible(false);
-		sfxButton.setVisible(false);
-		saveButton.setVisible(false);
-		quitButton.setVisible(false);
-		logButton.setVisible(false);
+
+		/*

```

Comment:
This may help http://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste

git rm --cached -r mydirectory